CardView: Update demo descriptions (#30108)
DevExpress/DevExtreme: apps/demos/Demos/DataGrid/Overview/jQuery/description.md

> DevExtreme DataGrid is a client-side grid control available as a jQuery component. This control supports binding to data from local arrays, JSON files, Web API and OData services, as well as custom remote services. Check out other demos in this section to explore more DataGrid features, like data editing and validation, paging, scrolling, etc.

Reviewer: ```suggestion
DevExtreme DataGrid is a client-side grid control available as a jQuery component. This control supports binding to data from local arrays, JSON files, Web API, OData services, and custom remote services. Check out other demos in this section to explore more DataGrid features, like data editing and validation, paging, scrolling, and so on.
```
